auth/README.md

```diff
@@ -353,6 +353,22 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 Twitter also returns an AuthToken Secret which can be accessed with `idpResponse.getIdpSecret()`.
 
+##### User metadata
+
+While `IdpResponse` provides user information about a specific sign-in instance, it is usually
+preferable to find the user name, email, and other metadata directly from the currently signed-in
+`FirebaseUser` instance (`auth.getCurrentUser()`). For example, you could determine if the user
+who just signed in is an existing or new one by comparing the user's creation and last sign-in time:
+
+```java
+FirebaseUserMetadata metadata = auth.getCurrentUser().getMetadata();
+if (metadata.getCreationTimestamp() == metadata.getLastSignInTimestamp()) {
```

Comment:
@samtstern There are no documented examples of how to use these methods, but I would assume this works, right?